docs/error-reporting/advanced/custom-symbol-server.md

```diff
@@ -14,9 +14,10 @@ Backtrace integrations using minidumps require symbol files to generate valuable
 To meet Backtrace's requirements, the custom symbol server must:
 
 - Follow Microsoft's standard symbol structure (that is, `<url>/<object_name>/<debug_id>/<file>`).
-- Use tools like Windows symstore to generate files in the required structure.
+-- Use tools like Windows symstore to generate files in the required structure.
```

Comment:
Double dashes is not an indented bullet, which I think is what you are trying to achieve? This is the result of double dashes.
![image](https://github.com/saucelabs/sauce-docs/assets/115846221/600ccc5d-edaa-4be3-86b3-03e64a18390e)

Use spacespacedash for an indented bullet.
![image](https://github.com/saucelabs/sauce-docs/assets/115846221/4e6cc166-35cf-4110-85c0-e2ec58bd859d)

```suggestion
  - Use tools like Windows symstore to generate files in the required structure.
```